src/get-city.ts

```
@@ -3,58 +3,67 @@ import { mapToCondition } from "./condition-mapping";
 import { mapToSVG } from "./icon-mapping";
 import { ICity } from "./interfaces/city";
 
-
-export function displayCityInfo(cityInfo : ICity, map : Map, userLocation = false) {
+export function displayCityInfo(
+  cityInfo: ICity,
+  map: Map,
+  userLocation = false,
+) {
   const existingWeatherDivs =
-    document.getElementsByClassName("previewInitWeather");
+    document.getElementsByClassName("weather-day-info");
   if (existingWeatherDivs.length > 0) {
-    const cityNameElement = document.getElementById("cityName")
-    if (!cityNameElement)
-    return null
-      cityNameElement.remove();
+    const cityNameElement = document.getElementById("cityName");
+    if (!cityNameElement) return null;
```

Good safety check. Maybe throw an error instead of returning null? A second opinion is needed on this.

---

"return null" has now been replaced with a thrown error. I am still open to suggestions on how to handle that scenario. Thank you!